contributing/development/configuring_an_ide/rider.rst

```diff
@@ -47,23 +59,31 @@ if you want to debug the editor, you need to configure the debugger first.
 .. figure:: img/rider_configurations.webp
    :align: center
 
-- Delete what is under **Program arguments** and paste the following line:
-
-::
+- Ensure the following values for the C++ Project Run Configuration:
 
-  -e --path <path to the Godot project>
+    - Exe Path : ``$(LocalDebuggerCommand)``
+    - Program Arguments: ``-e --path <path to the Godot project>``
+    - Working Directory: ``$(LocalDebuggerWorkingDirectory)``
+    - Before Launch has a value of "Build Project"
 
 This will tell the executable to debug the specified project without using the project manager.
 Use the root path to the project folder, not ``project.godot`` file path.
 
 .. figure:: img/rider_configurations_changed.webp
    :align: center
 
-- Finally click on apply and ok to save the changes.
+- Finally click on "Apply" and "OK" to save the changes.
 
-- You can now launch the debugger by pressing the **Debug** icon at the top of the screen and it will open the project to debug.
+- When you press the **Debug** icon at the top of the screen, JetBrains Rider will launch the Godot editor with the debugger attached.
 
 Alternatively you can use **Run > Attach to Process** to attach the debugger to a running Godot instance.
 
 .. figure:: img/rider_attach_to_process.webp
    :align: center
+
+- You can find the Godot instance by searching for ``godot.editor`` and then clicking ``Attach with LLDB``
+
+.. figure:: img/rider_attach_to_process_dialog.webp
+   :align: center
+
+Please consult the `JetBrains Rider <https://www.jetbrains.com/rider/documentation/>`_ documentation for any specific information about the JetBrains IDE.
```

Comment:
I think the issue is that you use the same name here, maybe:
```suggestion
Please consult the JetBrains Rider `documentation <https://www.jetbrains.com/rider/documentation/>`_ for any specific information about the JetBrains IDE.
```

Reply:
Thank you @AThousandShips . I've fixed the issue and submitted another change.